python/tskit/trees.py

```diff
@@ -2187,6 +2191,8 @@ def newick(self, precision=14, root=None, node_labels=None):
         :param dict node_labels: If specified, show custom labels for the nodes
             that are present in the map. Any nodes not specified in the map will
             not have a node label.
+        :param branch_lengths: If True (default), output branch lengths in the Newick
```

Comment:
Hmm, looking at this again in the presence of the other parameters I wonder if it's a bit misleading and it seems like we're providing a way to specify what hte branch lenghts *are*. Perhaps change to ``include_branch_lengths``? Probably better to be explicit.

Reply:
> The double underscore was deliberate I think, as a very-private indeed embarrassing recursive function.

I thought that double underscore didn't mean "very private", but instead marked it for name mangling: https://dbader.org/blog/meaning-of-underscores-in-python

Reply:
Yes, that's what happens in practice.